Source/JavaScriptCore/inspector/remote/cocoa/RemoteInspectorCocoa.mm

```diff
@@ -518,6 +518,17 @@ static bool canAccessWebInspectorMachPort()
         [listing setObject:target.nameOverride().createNSString().get() forKey:WIROverrideNameKey];
         [listing setObject:WIRTypeWebPage forKey:WIRTypeKey];
         break;
+    case RemoteInspectionTarget::Type::WebAssembly:
+        [listing setObject:target.url().createNSString().get() forKey:WIRURLKey];
+        [listing setObject:target.name().createNSString().get() forKey:WIRTitleKey];
+        [listing setObject:target.nameOverride().createNSString().get() forKey:WIROverrideNameKey];
+        [listing setObject:WIRTypeWebAssembly forKey:WIRTypeKey];
+        if (auto wcpPID = target.webContentProcessPID()) {
```

Comment:
Nit: I think the variable can just be `pid` since we get it out of `webContentProcessPID()` and then store it back into the same key.